### ISSUE TITLE
Question: How to  visualize  the tree that  the way  get?

### ISSUE THREAD

**Issue by LittleBoyHack** [kimhc6028/soft-decision-tree#3]
I want to visualize the tree to look how it stimulate the NN , just like the  picture  in  img  folder.  But  the source code seems  to do not have this function ?  what should  I  do?

**Comment by kimhc6028** [comment]
Yes, image is from the paper. I forgot most of the code, but I think each node class outputs computational results after computing forward(). So, get the results, and reshape them and you can get the processed image. Then you can visualize them with other python packages for visualization.

**Comment by lmartak** [comment]
hey @LittleBoyHack if it's still relevant, here's a piece of code that does exactly what you're discussing on my `tf.keras` implementation of the tree, so you can at least get inspiration for how to visualize the tree model filters: https://github.com/lmartak/distill-nn-tree
